src/screens/poa-form/index.js

```diff
@@ -426,6 +427,56 @@ class PoAForm extends React.Component {
       <Section>
         {disclaimer}
         <Heading tag='h1'>POWER OF ATTORNEY FOR CARE OF A MINOR CHILD</Heading>
+        <div>
+
+        <Stepper 
+          steps={ 
+            [
+              {
+                title: 'Child Information',
+                onClick: (e) => {
+                  e.preventDefault();
+                  this.renderStepOne();
+                  this.setState((state) => ({ step: 0 }));
+                }
+              },
+              {
+                title: 'Guardian Information',
```

Comment:
You need to update step 2 to include both Mother and Father parental information.  Right now it just includes Mother.